src/api/flaskr/common/config.py

```
@@ -1,47 +1,1030 @@
+        return "\n".join(lines)
+
+
+# Global instance
+__INSTANCE__ = None
+__ENHANCED_CONFIG__ = EnhancedConfig(ENV_VARS)
 
 
 class Config(FlaskConfig):
+    """Flask configuration wrapper with enhanced environment variable support."""
+
     def __init__(self, parent: FlaskConfig, app: Flask, defaults: dict = {}):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)